src/main/webapp/resources/theme1/src/facebook.js

```diff
@@ -76,9 +76,11 @@ window.fbAsyncInit = function() {
 }(document, 'script', 'facebook-jssdk'));
 
 //  See statusChangeCallback() for when this call is made.
-function callFbGraphApi(accessToken) {
+function callFbGraphApi(fbAccessToken) {
```

Comment:
this was very unclear what access token this was, so lets try to name the variable clearly